CONTRIBUTING.md

```diff
@@ -82,13 +84,41 @@ Please be as descriptive as possible when creating an issue. Give us the informa
 - Make sure you have added the necessary tests for your changes.
 - Make sure you have documented your changes, if necessary.
 
-### Submitting changes as pull Requests
+### Submitting changes as Pull Requests
 
 - Push your changes to a topic branch in the fork of your repository. If you are working locally, do this with `git push -u my-gh-repo topic/my-fix`. `my-gh-repo` is typically `origin`; check with `git remote -v`.
 - Submit a pull request to the SuperCollider repository.
 - The core team looks at pull requests on a regular basis in a weekly meeting that we hold in a public meeting. The hangout is announced in the weekly status updates that are sent to the sc-dev list.
 - You may receive feedback and requests for changes. We expect changes to be made in a timely manner. We may close the pull request if it isn't showing any activity.
 
+### Updating your fork
+In order to keep your fork up-to-date, you need to point it to the main SuperCollider repository. This is done by adding the main repository as another *remote*, usually called *upstream*. **Please note:** naming the main repository *upstream* is just a convention, not a requirement. If you already have a differently named remote pointing to the main SuperCollider repository, you can use that name instead of *upstream*.
+- If you have **not** yet created the *upstream* remote, create if first:
```

Comment:
if -> it. and just a tone thing, i think it looks a little cleaner not to bold/italicize too many words. I think "please note" above is fine, but "not" here and below, and "Be sure to have...!" don't need to be bolded, the wording is emphatic enough as it is. Others are free to chime in about this, it's just my preference and what I like to see in other documentation.

Also I believe that the convention in our documentation is to use code-style (with backticks) instead of italics for names like `remote` and `upstream`.